crowbar_framework/config/repos-ses.yml

```diff
@@ -41,25 +41,25 @@ suse-12.1:
       url:
       smt_path: "SUSE/Updates/SLE-HA/12-SP1/x86_64/update"
       ask_on_error: false
-    suse-enterprise-storage-3-pool:
-      name: "SUSE-Enterprise-Storage-3-Pool"
+    suse-enterprise-storage-4-pool:
+      name: "SUSE-Enterprise-Storage-4-Pool"
       required: "optional"
       features: ["ceph"]
-      repomd:
-        tag: "obsproduct://build.suse.de/SUSE:SLE-12-SP1:Update:Products:SES3/ses/3/POOL/x86_64"
-        key_md5: "9a62177e1c6852d48453ed3909956d6b"
```

Comment:
Oh, I guess we didn't do the change for this file, but if you look at the repos-cloud.yml file, we moved to fingerprints. So you shouldn't need anymore to remove things like you did.

Maybe move to fingerprints here there?